internal/server/firm_deputy_hub.go

```diff
@@ -1,22 +1,25 @@
 package server
 
 import (
-	"github.com/ministryofjustice/opg-sirius-supervision-firm-deputy-hub/internal/sirius"
 	"net/http"
 	"strconv"
 	"strings"
+
+	"github.com/ministryofjustice/opg-sirius-supervision-firm-deputy-hub/internal/sirius"
 )
 
 type FirmHubInformation interface {
 	GetFirmDetails(sirius.Context, int) (sirius.FirmDetails, error)
 }
 
 type firmHubVars struct {
-	Path        string
-	XSRFToken   string
-	Error       string
-	Errors      sirius.ValidationErrors
-	FirmDetails sirius.FirmDetails
+	Path           string
+	XSRFToken      string
+	Error          string
+	Errors         sirius.ValidationErrors
+	FirmDetails    sirius.FirmDetails
+	Success        bool
```

Comment:
Not sure I see the point of having a `Success` boolean and a `SuccessMessage`. The presence of a message is the indication of success.